TombEngine/Game/Lara/lara_struct.h

```diff
@@ -1316,54 +1315,54 @@ struct PlayerInventoryData
 	unsigned int TotalSecrets;
 
 	bool HasBinoculars = false;
-	bool HasCrowbar	   = false;
-	bool HasTorch	   = false;
+	bool HasCrowbar = false;
+	bool HasTorch = false;
 	bool HasLasersight = false;
-	bool HasSilencer   = false; // TODO: Unused.
-
-	int Puzzles[NUM_PUZZLES]			= {};
-	int Keys[NUM_KEYS]					= {};
-	int Pickups[NUM_PICKUPS]			= {};
-	int Examines[NUM_EXAMINES]			= {};
-	int PuzzlesCombo[NUM_PUZZLES * 2]	= {};
-	int KeysCombo[NUM_KEYS * 2]			= {};
-	int PickupsCombo[NUM_PICKUPS * 2]	= {};
+	bool HasSilencer = false; // TODO: Unused.
+
+	int Puzzles[NUM_PUZZLES] = {};
+	int Keys[NUM_KEYS] = {};
+	int Pickups[NUM_PICKUPS] = {};
+	int Examines[NUM_EXAMINES] = {};
+	int PuzzlesCombo[NUM_PUZZLES * 2] = {};
+	int KeysCombo[NUM_KEYS * 2] = {};
+	int PickupsCombo[NUM_PICKUPS * 2] = {};
 	int ExaminesCombo[NUM_EXAMINES * 2] = {};
 };
 
 struct LaraInfo
 {
 	static constexpr auto TARGET_COUNT_MAX = 16;
 
-	PlayerContext		Context	  = PlayerContext();
-	PlayerControlData	Control	  = {};
-	PlayerStatusData	Status	  = {};
-	PlayerEffectData	Effect	  = {};
+	PlayerContext		Context = PlayerContext();
+	PlayerControlData	Control = {};
+	PlayerStatusData	Status = {};
+	PlayerEffectData	Effect = {};
 	PlayerInventoryData Inventory = {};
 
 	// TODO: Move to PlayerControlData.
 	FlareData		  Flare = {};
 	TorchData		  Torch = {};
 	CarriedWeaponInfo Weapons[(int)LaraWeaponType::NumWeapons] = {}; // TODO: Move to WeaponControlData.
 
-	EulerAngles ExtraHeadRot	= EulerAngles::Identity;
-	EulerAngles ExtraTorsoRot	= EulerAngles::Identity;
+	EulerAngles ExtraHeadRot = EulerAngles::Identity;
+	EulerAngles ExtraTorsoRot = EulerAngles::Identity;
 	EulerAngles TargetArmOrient = EulerAngles::Identity;
-	ArmInfo		LeftArm			= {};
-	ArmInfo		RightArm		= {};
+	ArmInfo		LeftArm = {};
+	ArmInfo		RightArm = {};
 
 	ItemInfo* TargetEntity = nullptr; // TargetEntityPtr. Should use item number instead?
-	std::array<ItemInfo*, TARGET_COUNT_MAX> TargetList	= {};
+	std::array<ItemInfo*, TARGET_COUNT_MAX> TargetList = {};
 	std::array<ItemInfo*, TARGET_COUNT_MAX> LastTargets = {};
 
 	// TODO: Rewrite and restore spasm effect. Also move to PlayerEffectData?
-	int HitFrame	 = 0; // Frame index.
+	int HitFrame = 0; // Frame index.
 	int HitDirection = 0; // Cardinal direction.
 
 	// Item number? Only ever set to NO_VALUE or 1. Probably anim object ID. Might not be needed since AnimObjectID is kept in item.Animation.
 	int ExtraAnim = 0;
 
-	signed char Location		= 0;
+	signed char Location = 0;
 	signed char HighestLocation = 0;
-	signed char LocationPad		= 0;
+	signed char LocationPad = 0;
```

Comment:
Apparently all indentation was messed up in this file, please restore it to original state.

Reply:
Have restored the original file.